src/Compilers/VisualBasic/Portable/BoundTree/BoundTreeVisitor.vb

```diff
@@ -123,6 +123,7 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
         Protected Sub New()
         End Sub
 
+        <DebuggerHidden>
```

Comment:
This and the one on DefaultVisit bring VB in line with C#: we already had DebuggerHidden attributes on the equivalent methods there.